java/maven/test/unit/data/projects/multiproject/democa/oci/pom_add_artifact.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.exampleca</groupId>
+  <artifactId>oci</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>${packaging}</packaging>
+  <name>democa-${artifactId}</name>
+
+  <parent>
+    <groupId>com.exampleca</groupId>
+    <artifactId>democa-parent</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <packaging>jar</packaging>
+    <jdk.version>17</jdk.version>
+    <release.version>17</release.version>
+    <micronaut.version>4.0.3-oracle-00001</micronaut.version>
+    <micronaut.test.resources.enabled>true</micronaut.test.resources.enabled>
+    <micronaut.runtime>netty</micronaut.runtime>
+    <exec.mainClass>com.exampleca.Application</exec.mainClass>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>central</id>
+      <url>https://repo.maven.apache.org/maven2</url>
+    </repository>
+    
+    <repository>
+      <id>gcn</id>
+      <url>https://maven.oracle.com/public</url>
+    </repository>
```

Comment:
can this be reproduced without requiring two repos and so many dependencies?

unit tests shouldn't download entire demo applications if it is somehow avoidable.